### PR TITLE
feat(content-block-cards): create web component

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#3782 

### Description

create content-block-cards web component

### Changelog

**New**

- component, tests, stories

### CODE REVIEW

File: packages/web-components/src/components/content-block-cards/__stories__/content-block-cards.stories.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    <div slot="heading">Natural Language Processing.</div>
+    <dds-card-cta-footer slot="footer">
+      ${ArrowRight20({ slot: 'icon' })}
+    </dds-card-cta-footer>
+  </dds-card-group-item>
+`;
+
+const cardGroupItemWithVideos = html`
+  <dds-card-group-item cta-type="video" href="1_9h94wo6b">
+    Test Video - 1:1
```
Comment (asudoh) [inline packages/web-components/src/components/content-block-cards/__stories__/content-block-cards.stories.ts:62]: Does something like https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/13b0181/packages/web-components/src/components/content-group-simple/__stories__/content-group-simple.stories.ts#L192 help?

**Review by asudoh — COMMENTED** [review]
Great progress - Thanks @ariellalgilmore!

File: packages/web-components/src/components/content-block-cards/__stories__/README.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      <dds-card-cta-footer slot="footer">
+        ${ArrowRight20({ slot: 'icon' })}
+      </dds-card-cta-footer>
+    </dds-card-group-item>
+  </dds-card-group>
+  <dds-card-cta slot="cta" cta-type="${ifNonNull(ctaType)}" href="${ifNonNull(href)}">
+    <p>${ctaCopy}</p>
+    ${ArrowRight20({ slot: 'footer' })}
+  </dds-card-cta>
+</dds-content-block-cards>
```
[the remark this answers is not in the document]
Comment (asudoh) [reply packages/web-components/src/components/content-block-cards/__stories__/README.stories.mdx]: Update: @ariellalgilmore You can search for `${` to see where to fix.

File: packages/web-components/examples/codesandbox/components/content-block-cards/index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                  Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+                </p>
+                <dds-card-cta-footer slot="footer">
+                  ${ArrowRight20({ slot: 'icon' })}
+                </dds-card-cta-footer>
+              </dds-card-group-item>
+            </dds-card-group>
+            <dds-card-cta slot="cta" cta-type="${ifNonNull(ctaType)}" href="${ifNonNull(href)}">
+              <p>${ctaCopy}</p>
+              ${ArrowRight20({ slot: 'footer' })}
```
Comment (asudoh) [inline packages/web-components/examples/codesandbox/components/content-block-cards/index.html:62]: Do you want to eliminate the lit-html syntax from this file, too?

**Review by asudoh — COMMENTED** [review]
Just one thing - Thanks @ariellalgilmore!

File: packages/web-components/examples/codesandbox/components/content-block-cards/index.html (hunk 3 above)
Comment (ariellalgilmore) [reply packages/web-components/examples/codesandbox/components/content-block-cards/index.html]: Thanks!!! Updated!

**Review by asudoh — APPROVED** [review]
Thanks @ariellalgilmore for all the updates!

**Pull request merged**